WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.3.0"
+  s.version       = "1.3.1-beta.1"
```

Comment:
Is it worth calling this `1.4.0-beta.1` since it has a major version change to WPKit?

Reply:
Ok let me fix it!

Reply:
@jtreanor done!